apps/data-scanner/lib/models/phones/Phones.mjs

```diff
@@ -0,0 +1,38 @@
+import ModelBase from '../ModelBase.mjs';
+
+export class Phones extends ModelBase {
+    /**
+     * @typedef {Class} Phones
+     * @property getAddingList
+     * @property getRemovingList
+     * @property updateSession
+     */
+
+    /**
+     * @method
+     * @returns {Promise<Object>}
+     */
+    async getAddingList() {
+        const phones = await this.repository.phones.read({ tracked: true });
+        return phones.filter((phone) => !phone.session_id);
```

Comment:
implement filtering on sql level. pass one more parameter to read method({ tracked: true, session: true })

Reply:
done